### PR TITLE
Text min-width in edit mode

### PR DESCRIPTION
**Pull request by seaneking**
Text should be as wide as placeholder when visible (ie: edit mode with no content)

### CODE REVIEW

**Comment by bedeoverend** [comment]
Is this overriding the users' styles? We're setting the `min-width` as placeholder changes, meaning that if the user has applied a min-width to the `simpla-text` element it'll be lost.

**Comment by bedeoverend** [comment]
Also, it'd be better if togglePlaceholderWidth weren't called [here](https://github.com/simplaio/simpla-text/pull/17/files#diff-8563184cdc3681d4ef2d0ef24fd1407fR150) and instead was called on changed to `usePlaceholder` via an observer e.g. 
```js
this.observers = [
 '_togglePlaceholderWidth(usePlaceholder)'
]
```

**Comment by seaneking** [comment]
@bedeoverend good for me to merge?

File: src/simpla-text/simpla-text.js [hunk 1]
```diff
@@ -153,6 +150,23 @@ class SimplaText {
...
+   */
+  _usePlaceholderChanged(usePlaceholder) {
+    let placeholderWidth = this.$.placeholder.getBoundingClientRect().width;
+
+    if (!this.__minWidth) {
+      this.__minWidth = getComputedStyle(this).minWidth;
+    }
+
+    if (this.__minWidth && this.__minWidth === '0px') {
+      console.log(usePlaceholder)
```
Comment (bedeoverend) [inline src/simpla-text/simpla-text.js]: Got a leftover console.log

**Comment by bedeoverend** [comment]
A part from the console.log, yeah LGTM

**Comment by seaneking** [comment]
Derp

**Pull request merged**